app/assets/stylesheets/modules/shared/email_status.scss

```diff
@@ -36,7 +42,7 @@
 
     &.last {
       img.approved.linear, img.pending.linear, img.rejected.linear {
-        background-image: none;
+        background-image: none !important;
```

Comment:
In order to bypass the roady-rails bug, I have to inline the styles directly in the markup. The inline styles are taking precedence over the css styles that remove the vertical line for the 'last' approver. Otherwise, it would show as follows:

![screen shot 2015-06-26 at 1 42 09 pm](https://cloud.githubusercontent.com/assets/11333/8384793/2cbcc054-1c09-11e5-89f9-4fdd03e20757.png)

I can probably pull the `last` class into the img tag... will try that.